vissl/trainer/train_zero_task.py

```diff
@@ -9,6 +9,7 @@
 from vissl.trainer.train_task import SelfSupervisionTask
 from vissl.utils.hydra_config import AttrDict
 
+
```

Comment:
nit: is this change intended? :)

Reply:
This change looks like it was introduced by `isort` to enforce PEP 8 two newlines